docs/getting-started-monitoring.md

```diff
@@ -118,6 +123,16 @@ First, you need to install the following tools used in this guide.
    helm install scalar-monitoring prometheus-community/kube-prometheus-stack -n monitoring -f scalar-prometheus-custom-values.yaml
    ```
 
+1. Add the `grafana` helm repository.
+  ```console
+  helm repo add grafana https://grafana.github.io/helm-charts
+  ```
```

Comment:
It seems that we need one more indent in this code snippet.

Reply:
Thanks for the catch.
I revised it in https://github.com/scalar-labs/helm-charts/pull/87/commits/e59d552166368a9f3258d2498872a1778e37043d